book/src/dev/release-process.md

```diff
@@ -0,0 +1,118 @@
+# Zebra versioning and releases
+
+This document contains the practices that we follow to provide you with a leading-edge application, balanced and with stability.
```

Comment:
seems like theres an extra "and" should this be: "provide you with a leading-edge application, balanced with stability." ?